apps/src/StudioApp.js

```diff
@@ -2107,7 +2107,8 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     config.dropletConfig,
     config.level.codeFunctions,
     config.level.autocompletePaletteApisOnly,
-    this.Dialog);
+    this.Dialog,
+    config.app);
```

Comment:
is this a safe way to get the application type? I've tested with gamelab/applab in projects and levels and it works, but nothing else seems to reference this.